backend/vkt/src/main/java/fi/oph/vkt/model/Person.java

```diff
@@ -46,6 +47,12 @@ public class Person extends BaseEntity {
   @Column(name = "date_of_birth")
   private LocalDate dateOfBirth;
 
+  @Column(name = "payment_link_hash")
+  private String paymentLinkHash;
+
+  @Column(name = "payment_link_expires")
+  private LocalDateTime paymentLinkExpires;
```

Comment:
Nää pitäis varmaan Enrollmentiin mieluummin sitoa?

Reply:
Hyvä pointti. 👍 Muutin enrollmentin alle.